README.md

```diff
@@ -52,6 +52,53 @@ feed is friendly or blocked.
 - Replication is strictly disabled for:
   - Any feed you explicitly block
 
+### Configuration
+
+Some parameters and opinions can be configured by the user or by application
+code through the conventional [ssb-config](https://github.com/ssbc/ssb-config)
+object. The possible options are listed below:
+
+````js
+{
+  replicationScheduler: {
+    /**
+     * If `partially` is an array, it tells the replication scheduler to perform
+     * partial replication, whenever remote feeds support it. The array
+     * describes the tree of meta feeds that we are interested in replicating.
+     * If `partially` is `false (which it is, by default), then all friendly
+     * feeds will be requested in full.
```

Comment:
@arj03 This is a fresh idea I just had, keen to hear feedback. Basically this config will specify everything that we want to replicate partially. Notice that it looks a lot like meta feed fields. But there is a small difference: `children` field exists, and not all these objects are in their "full shape", e.g. `query.type` exists but not `query.author`. The semantics would be that all these fields specified *must* be true but the fields omitted can be of any value, i.e. `query.author` can be anything, as well as `feedformat` can be anything because it's not specified.

Reply:
Overall I like this syntax, it enables the user to specify their own rules. Not sure about the leaving out author part. Maybe we can do something like @author? or something to specify the query more fully so we don't accidently replicate something other than expected. Also this enables reconfiguring and thus downloading a new part of the feed which is really nice, have to be careful there about if we already have existing data though. Last comment is about the name `partially`, for me seems to indicate that these things would be partially replicated, but it is the other way around, we replicate these in full in order to replicate the main feed partially.

Reply:
By the way, before we commit to this idea, I'm open to other suggestions that would be overall very different to this one, in the big picture. Like, what solution were you thinking for this, before I opened this branch?

> Not sure about the leaving out author part. Maybe we can do something like @author?

Hmm, how would this `{type: 'vote', author: '@author'}` work? What does it even mean? (Can we even know the main feedId from this syntax?)

> Last comment is about the name `partially`

Any suggestions?